server/apps/prepopulate/data_initialization/validators.json

```diff
@@ -41,10 +41,16 @@
         },
         "subject" : {
             "type" : "list",
-            "required" : true
+            "required" : true,
+            "minlength" : 1
```

Comment:
OK, if this is the way the validator is designed, then it's fine of course and my comment can be ignored (thanks for the link BTW).

:+1: